sdk/vaa/structs.go

```diff
@@ -197,6 +197,8 @@ func (c ChainID) String() string {
 		return "xpla"
 	case ChainIDBtc:
 		return "btc"
+	case ChainIDHedera:
```

Comment:
Also need to add stuff to structs_tests.go.

Reply:
Done